fastlane/Fastfile

```diff
@@ -283,6 +286,56 @@ platform :ios do
     trigger_release_build(branch_to_build: "release/#{version}")
   end
 
+  #####################################################################################
+  # update_metadata_on_app_store_connect
+  # -----------------------------------------------------------------------------------
+  # Uploads the localized metadata tracked in the project repository to App Store
+  # Connect, optionally updating screenshots, too.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_metadata_on_app_store_connect [with_screenshots:<with screenshots>]
+  #
+  # Example:
+  # bundle exec fastlane update_metadata_on_app_store_connect
+  # bundle exec fastlane update_metadata_on_app_store_connect with_screenshots:true
+  #####################################################################################
+  desc 'Updates the App Store Connect localized metadata'
+  lane :update_metadata_on_app_store_connect do |options|
+    update_wordpress_metadata_on_app_store_connect(options)
+    update_jetpack_metadata_on_app_store_connect(options)
+  end
+
+  #####################################################################################
+  # update_wordpress_metadata_on_app_store_connect
+  # -----------------------------------------------------------------------------------
+  # This lane reads the information from `fastlane/metadata/` and uploads it to
+  # App Store Connect for the version specified in the version xcconfig file.
+  #
+  # Optionally, it can also upload new screenshots, overriding the existing ones.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect
+  #
+  # Example:
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect
+  # bundle exec fastlane update_wordpress_metadata_on_app_store_connect with_screenshots:true
+  #####################################################################################
+  desc 'Uploads the WordPress metadata to App Store Connect, localized, and optionally including screenshots.'
+  lane :update_wordpress_metadata_on_app_store_connect do |options|
+    # Skip screenshots by default. The naming is "with" to make it clear that
+    # callers need to opt-in to adding screenshots. The naming of the deliver
+    # parameter, on the other hand, uses the skip verb.
+    with_screenshots = options.fetch(:with_screenshots, false)
+    skip_screenshots = with_screenshots == false
```

Comment:
👍  I had a doubt while reading this that the option would be interpreted as a String and that passing `with_screenshots:false` for example would make `with_screenshots` variable be `"false"` String and that the comparison would end up with `skip_screenshots = false`…
But, turns out, fastlane is smart and interprets the option as a boolean… and even better, it accepts other spelling of the truthy/falsy values, e.g calling the. lane with `with_screenshots:yes` also results in `skip_screenshots = false` and `with_screenshots:no` results in `skip_screenshots = true`, so 👌 🎉 

(Only nit I have is that I'd have written it `skip_screenshots = !with_screenshots`, but 🤷 )